docs/guides/01-mocking.md

```diff
@@ -165,7 +163,7 @@ Pet:
         x-faker: image.imageUrl
 ```
 
-When we call `curl http://127.0.0.1:4010/pets/123 -H "Prefer: dynamic=true"`, the operation references this component so we get a doggie:
+Call `curl http://127.0.0.1:4010/pets/123 -H "Prefer: dynamic=true"`, the operation references this component and a doggie is returned:
```

Comment:
This reads a little bit weird to me... Maybe we can say "Making the call:" instead?

Reply:
Done